sycl/test-e2e/ESIMD/lsc/lsc_gather_scatter_stateless_64.cpp

```diff
@@ -40,18 +40,26 @@ int main() {
          uint64_t offsetStart = (Size - VL) * sizeof(uint64_t);
          simd<uint64_t, VL> offset(offsetStart, sizeof(uint64_t));
          simd<uint64_t, VL> beginning(0, sizeof(uint64_t));
-         simd<uint64_t, VL> va = lsc_gather<uint64_t>(PA, beginning);
+         simd<uint64_t, VL> va =
+             lsc_gather<uint64_t, 1, lsc_data_size::default_size,
+                        cache_hint::none, cache_hint::none, VL>(PA, beginning);
          simd_mask<VL> pred = 1;
          simd<uint64_t, VL> old_values = 0;
          lsc_prefetch<uint64_t, 1, lsc_data_size::default_size,
-                      cache_hint::cached, cache_hint::cached>(PA, offset);
+                      cache_hint::cached, cache_hint::cached, VL>(PA, offset);
```

Comment:
Why VL is needed here? 'offset' is simd<uint64_t, VL> and 'VL' should be deduce-able.
If this case (and/or other cases like this) worked before and started failing at some point?

Reply:
I think yes, but that's an existing problem that I would prefer to solve in another PR, there might be some fallout or complicated cases.